### PR TITLE
Bump up netty to 4.1.99

### PR DESCRIPTION
**Pull request by kezhenxu94**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. NO.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

**Review by wu-sheng — APPROVED** [review]

**Comment by lujiajing1126** [comment]
As I've noticed from grpc-java repo, https://github.com/grpc/grpc-java/issues/10401, netty 4.1.94 suffers from performance issue.

**Comment by wu-sheng** [comment]
Interesting. @kezhenxu94 Is 4.1.93 good enough from CVE perspective?

File: pom.xml [hunk 1]
```diff
@@ -170,7 +170,7 @@
 
         <!-- core lib dependency -->
         <grpc.version>1.53.0</grpc.version>
-        <netty.version>4.1.86.Final</netty.version>
+        <netty.version>4.1.94.Final</netty.version>
```
Comment (lujiajing1126) [inline pom.xml:173]: What about 4.1.97. As it is used by grpc-java

https://github.com/grpc/grpc-java/commit/88b3484a602336b6688ca943d29b033c127dc533

**Review by lujiajing1126 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (wu-sheng) [reply pom.xml:173]: @lujiajing1126 We manage netty version separately AFAIK. We ignore the gRPC-Java dependency. The skywalking-eye license check verified the final use.

**Comment by kezhenxu94** [comment]
The regression is fixed in 4.1.96.Final and now i bump to 4.1.99.Final

**Review by lujiajing1126 — APPROVED** [review]

**Pull request merged**